qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -141,6 +141,7 @@ class FermionicOp(SecondQuantizedOp):
             fermion += FermionicOp(somedata)
 
     """
+    _truncate = 200
```

Comment:
Should we have some class level comment that talks about the str truncation and the default value truncation is set to? It might be helpful I think/